SYCL/Basic/cuda_max_wgs_error.cpp

```diff
@@ -0,0 +1,56 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// REQUIRES: cuda
+
+#include <numeric>
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+
+const size_t lsize = 32;
+const size_t max_x = (1ull << 31ull) - 1ull;
+const size_t max_yz = 65535;
```

Comment:
Instead of using hard-coded values, would it make sense to use the [`max_work_groups` query extension](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_max_work_group_query.md)? Though I don't imagine these values will change on NVIDIA devices for a while, it should hopefully make it a little more robust and potentially make it easier to enable the test for other backends in the future.

Reply:
done